src/core/Breadcrumb/Breadcrumb.tsx

```diff
@@ -12,15 +12,15 @@ import { BreadcrumbLink, BreadcrumbLinkProps } from './BreadcrumbLink';
 
 type BreadcrumbVariant = 'default' | 'link';
 
-export interface BreadcrumbProps extends CompBreadcrumbProps, ThemeComponent {
+export interface BreadcrumbProps extends CompBreadcrumbProps, TokensComponent {
   /**
    * 'default' | 'expansion'
    * @default default
    */
   variant?: BreadcrumbVariant;
 }
 
-const StyledBreadcrumb = styled(({ theme, ...passProps }: BreadcrumbProps) => (
+const StyledBreadcrumb = styled(({ tokens, ...passProps }: BreadcrumbProps) => (
```

Comment:
There seems to be a couple of instances alike with tokens (previously theme) destructured but never used.

Reply:
This was covered earlier today and makes sense now. Additional props are destructured to strip those from passProps. We need to update linting rules to allow this though.